app/system/views/mailpartials/form/form_tabs.blade.php

```diff
@@ -0,0 +1,53 @@
+@php
+    $type = $tabs->section;
+    $activeTab = $activeTab ? $activeTab : '#'.$type.'tab-1';
+@endphp
+<div class="tab-heading">
+    <ul class="form-nav nav nav-tabs">
+        @php
+            $index = 0;
+        @endphp
+        @foreach ($tabs as $name => $fields)
+            @php
+                $index++;
+                $tabName = '#'.$type.'tab-'.$index;
+            @endphp
+            <li class="nav-item">
+                <a
+                    class="nav-link{{ ($tabName == $activeTab) ? ' active' : '' }}"
+                    href="{{ $tabName }}"
+                    data-toggle="tab"
+                >@lang($name)</a>
+            </li>
+        @endforeach
+    </ul>
+</div>
+
+<div class="row no-gutters">
+    <div class="col-md-8">
+        <div class="tab-content">
+            @php
+                $index = 0;
+            @endphp
+            @foreach ($tabs as $name => $fields)
+                @php
+                    $index++;
+                    $tabName = '#'.$type.'tab-'.$index;
+                @endphp
+                <div
+                    class="tab-pane {{ ($tabName == $activeTab) ? 'active' : '' }}"
+                    id="{{ $type.'tab-'.$index }}">
+                    <div class="form-fields">
+                        {!! $this->makePartial('form/form_fields', ['fields' => $fields]) !!}
+                    </div>
+                </div>
+            @endforeach
```

Comment:
use `$loop->iteration` instead of `$index` and avoid using the `@php` directive except where necessary